sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/EditContactSteps.java

```diff
@@ -146,9 +511,99 @@ public Contact collectContactData() {
+            webDriverHelpers.getValueFromWebElement(IDENTIFICATION_SOURCE_DETAILS_COMBOBOX_INPUT))
+        .identificationSourceDetails(identificationSource)
+        .typeOfContact(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(TYPE_OF_CONTACT_OPTIONS))
+        .additionalInformationOnContactType(
+            webDriverHelpers.getValueFromWebElement(
+                ADDITIONAL_INFORMATION_OF_THE_TYPE_OF_CONTACT_INPUT))
+        .contactCategory(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(CONTACT_CATEGORY_OPTIONS))
+        .relationshipWithCase(
+            webDriverHelpers.getValueFromWebElement(RELATIONSHIP_WITH_CASE_COMBOBOX_INPUT))
         .descriptionOfHowContactTookPlace(
             webDriverHelpers.getValueFromWebElement(DESCRIPTION_OF_HOW_CONTACT_TOOK_PLACE_INPUT))
+        .prohibitionToWork(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(PROHIBITION_TO_WORK_OPTIONS))
+        .homeBasedQuarantinePossible(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+                HOME_BASED_QUARANTINE_OPTIONS))
+        .quarantine(webDriverHelpers.getValueFromWebElement(QUARANTINE_COMBOBOX_INPUT))
+        .highPriority(webDriverHelpers.getValueFromWebElement(HIGH_PRIORITY_CHECKBOX))
+        .diabetes(webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(DIABETES_OPTIONS))
+        .immunodeficiencyIncludingHiv(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(HIV_OPTIONS))
+        .liverDisease(webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(LIVER_OPTIONS))
+        .malignancy(webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(MALIGNANCY_OPTIONS))
+        .chronicPulmonaryDisease(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(PULMONARY_OPTIONS))
+        .renalDisease(webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(RENAL_OPTIONS))
+        .chronicNeurologicalNeuromuscularDisease(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(NEUROLOGIC_OPTIONS))
+        .cardiovascularDiseaseIncludingHypertension(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(CARDIOVASCULAR_OPTIONS))
+        .additionalRelevantPreexistingConditions(
+            webDriverHelpers.getValueFromWebElement(ADDITIONAL_RELEVANT_PRE_CONDITIONS_TEXT))
+        .vaccinationStatusForThisDisease(
+            webDriverHelpers.getValueFromWebElement(VACCINATION_STATUS_COMBOBOX_INPUT))
+        .immunosuppressiveTherapy(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+                IMMUNOSUPPRESSIVE_THERAPY_OPTIONS))
+        .activeInCare(
+            webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(CARE_OVER_60_OPTIONS))
+        .overwriteFollowUp(webDriverHelpers.getValueFromWebElement(OVERWRITE_FOLLOW_UP_CHECKBOX))
+        // .dateOfFollowUpUntil(parsedDateOfFollowUp)
```

Comment:
Please remove commented code

Reply:
commented code is required to run when the issue tagged will be fixed. see the feature file.